code/DDSCodeTester.cpp

```diff
@@ -3128,6 +3128,30 @@ void xml_profiles_examples()
         }
         //!--
     }
+        {
+        std::string custom_name;
```

Comment:
The idea of this line and the following is to showcase the situation in which you would need this function. The first one being a variable value obtained previously and another which is hard coded. For this I think it is better to not show how the custom name is obtained as it would only make the example more cluttered.